# Pololu based motors

#Propulsao
prop_one:
  pololu_id: 12
  motor_id: 0
  init: 1500
  min: 1000
  max: 2000
  reverse: true
  calibration:
    min_pulse: 1000
    min_angle: -45
    max_pulse: 2000
    max_angle: 45

prop_two:
  pololu_id: 12
  motor_id: 1
  init: 1500
  min: 1000
  max: 2000
  reverse: true
  calibration:
    min_pulse: 1000
    min_angle: -45
    max_pulse: 2000
    max_angle: 45

prop_three:
  pololu_id: 12
  motor_id: 2
  init: 1500
  min: 1000
  max: 2000
  reverse: true
  calibration:
    min_pulse: 1000
    min_angle: -45
    max_pulse: 2000
    max_angle: 45

prop_four:
  pololu_id: 12
  motor_id: 3
  init: 1500
  min: 1000
  max: 2000
  reverse: true
  calibration:
    min_pulse: 1000
    min_angle: -45
    max_pulse: 2000
    max_angle: 45


#Vetorizacao
vet_one:
  pololu_id: 12
  motor_id: 6
  init: 1500
  min: 1000
  max: 2000
  reverse: true
  calibration:
    min_pulse: 1000
    min_angle: -45
    max_pulse: 2000
    max_angle: 45

vet_two:
  pololu_id: 12
  motor_id: 7
  init: 1500
  min: 1000
  max: 2000
  reverse: true
  calibration:
    min_pulse: 1000
    min_angle: -45
    max_pulse: 2000
    max_angle: 45

vet_three:
  pololu_id: 12
  motor_id: 8
  init: 1500
  min: 1000
  max: 2000
  reverse: true
  calibration:
    min_pulse: 1000
    min_angle: -45
    max_pulse: 2000
    max_angle: 45

vet_four:
  pololu_id: 12
  motor_id: 9
  init: 1500
  min: 1000
  max: 2000
  reverse: true
  calibration:
    min_pulse: 1000
    min_angle: -45
    max_pulse: 2000
    max_angle: 45

#Leme
leme_one:
  pololu_id: 12
  motor_id: 12
  init: 1500
  min: 1000
  max: 2000
  reverse: true
  calibration:
    min_pulse: 1000
    min_angle: -45
    max_pulse: 2000
    max_angle: 45

leme_two:
  pololu_id: 12
  motor_id: 13
  init: 1500
  min: 1000
  max: 2000
  reverse: true
  calibration:
    min_pulse: 1000
    min_angle: -45
    max_pulse: 2000
    max_angle: 45

leme_three:
  pololu_id: 12
  motor_id: 14
  init: 1500
  min: 1000
  max: 2000
  reverse: true
  calibration:
    min_pulse: 1000
    min_angle: -45
    max_pulse: 2000
    max_angle: 45

leme_four:
  pololu_id: 12
  motor_id: 15
  init: 1500
  min: 1000
  max: 2000
  reverse: true
  calibration:
    min_pulse: 1000
    min_angle: -45
    max_pulse: 2000
    max_angle: 45

#Digital Outputs

digital_o_one:
  pololu_id: 12
  motor_id: 22
  init: 1499
  min: 1000
  max: 2000
  reverse: false
  calibration:
    min_pulse: 1000
    min_angle: -500
    max_pulse: 2000
    max_angle: 500

digital_o_two:
  pololu_id: 12
  motor_id: 23
  init: 1499
  min: 1000
  max: 2000
  reverse: false
  calibration:
    min_pulse: 1000
    min_angle: -500
    max_pulse: 2000
    max_angle: 500

#Digital Inputs

digital_i_one:
  pololu_id: 12
  motor_id: 18
  init: 1499
  min: 1000
  max: 2000
  reverse: false
  calibration:
    min_pulse: 1000
    min_angle: -500
    max_pulse: 2000
    max_angle: 500





